src/testing.rs

```diff
@@ -13,12 +17,19 @@ pub struct ProposerRewardsExecutor {
     excluded_proposals_path: Option<String>,
     active_voteplan_path: Option<String>,
     challenges_path: Option<String>,
-    vit_station_url: String,
+    vit_station_url: Option<String>,
 }
 
-impl Default for ProposerRewardsExecutor {
+#[derive(Debug, Error)]
+pub enum Error {
+    #[error(transparent)]
+    Io(#[from] std::io::Error),
+}
+
+impl Default for ProposerRewardsCommand {
     fn default() -> Self {
         Self {
+            python_exec: "python3".to_string(),
```

Comment:
This requires python3 to be in path, not sure of how common that is, but I would at least put a disclaimer somewhere that it's needed for testing